lib/graphql/resolvers.ts

```diff
@@ -118,6 +122,7 @@ const resolvers = {
     // Reports
     generateApplicationsReport: authorize(generateApplicationsReport, ['SECRETARY']),
     generatePermitHoldersReport: authorize(generatePermitHoldersReport, ['SECRETARY']),
+    generateAccountantReport: authorize(generateAccountantReport, [`SECRETARY`]),
```

Comment:
```suggestion
    generateAccountantReport: authorize(generateAccountantReport, [`ACCOUNTING`]),
```
This feature is for accounting team only